<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalkTogether</class>
 <widget class="QMainWindow" name="TalkTogether">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TalkTogether</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="disconnectBtn">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>10</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 1px solid rgb(24, 103, 155);
border-radius: 8px;
background-color: rgb(124, 203, 255);
color: white;
width: 200px;
height: 30px
}

QPushButton:hover {
	color: balck;
	background-color:  rgb(64, 93, 255);
}

</string>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusL">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Online:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="msgDisplayer">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>790</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	border: 1px solid black;
	border-radius: 8px;
	padding-left: 5px;
	padding-right: 5px;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputMsg">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>560</y>
      <width>790</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid black;
	border-radius: 8px;
	padding-left: 5px;
	padding-right: 5px;
	height: 30px;
}</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="maxLength">
     <number>2048</number>
    </property>
    <property name="placeholderText">
     <string>Message</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMsgBtn">
    <property name="geometry">
     <rect>
      <x>763</x>
      <y>562</y>
      <width>30</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius: 8px;
width: 30px;
height: 30px;
}

QPushButton:hover {
	background-color:  rgb(185, 233, 255);	
}

QPushButton:pressed {
	background-color:  rgb(185, 233, 255);	
}

QPushButton:default {
	border-color: navy;
}</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset resource="../resurce/Resource.qrc">
      <normaloff>:/icons/send.png</normaloff>:/icons/send.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resurce/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
